test/automated-browser-tests.js

```diff
@@ -59,16 +60,23 @@ describe('Perform Browser Tests', function() {
         `${testServerURL}/test/browser-tests/`
       )
       .then(testResults => {
-        automatedBrowserTesting.manageTestResults(
+        const errorMessage = mochaHelper.prettyPrintErrors(
           browserInfo.prettyName,
           testResults
         );
+        if (errorMessage) {
+          throw new Error(errorMessage);
+        }
       });
     });
   };
 
   const automatedBrowsers = automatedBrowserTesting.getAutomatedBrowsers();
   automatedBrowsers.forEach(browserInfo => {
+    if (browserInfo.releaseName === 'unstable') {
```

Comment:
Could there be a way of opting-in to testing against the unstable releases of browsers rather than hardcoding this bypass? I could imagine some scenarios where the tests are run outside of an automated build and we care about testing against Canary, etc.